DEVELOPING.md

```diff
@@ -17,7 +17,7 @@ You would only do this once after you cloned the repository.
 
 When you are ready to commit
 
-1.  We enforces commit message format. We recommend using [commitizen](https://github.com/commitizen/cz-cli) by installing it with `yarn add --global commitizen` then commit using `git cz` which will prompt you questions to format the commit message.
+1.  We enforce commit message format. We recommend using [commitizen](https://github.com/commitizen/cz-cli) by installing it with `yarn add commitizen --global` then commit using `git cz` which will prompt you questions to format the commit message.
```

Comment:
Isn't the right way to do this `yarn global add commitizen` now? Probably doesn't matter.

Reply:
That worked for me, but it's a dev dependency anyway so happy to update to whatever.